lib/github/get-accounts.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../logger');
+
+module.exports = async (subscriptions, robot) => {
+  const accounts = await Promise.all(subscriptions.map(async (subscription) => {
+    const github = await robot.auth(subscription.Installation.githubId);
+    try {
+      const account = await github.users.getById({ id: subscription.githubId });
+      return account.data;
+    } catch (err) {
+      logger.error({ err, accountId: subscription.githubId }, 'Could not find account for subscription');
```

Comment:
I don't believe we would ever get a 404 back from this because all GitHub accounts are public. So even with our installation token, we'll always get an account back here.

Reply:
What if an account is deleted?

Reply:
🙈 Yes, that's a very good point hahaha 😅 

I think I mentioned this because we often use the get-repo equivalent as a way to find out whether our current access token has access to a repository, but that doesn't apply in this case. My bad! 😆

Reply:
No problem! 😉